extensions/VisualEditor/wikia/modules/ve/init/ve.init.wikia.ViewPageTarget.js

```diff
@@ -442,28 +477,56 @@ ve.init.mw.ViewPageTarget.prototype.loadAndRenderFancyCaptcha = function ( $cont
 	} );
 };
 
-/**
- * Get the licensing
- *
- * @returns {jQuery}
- */
-ve.init.wikia.ViewPageTarget.prototype.getLicense = function () {
-	if ( !this.$license ) {
-		this.$license = this.$('<div>')
-			.append(
-				this.$( '<p>' ).addClass( 've-ui-wikia-license' )
-					.html( ve.init.platform.getParsedMessage( 'copyrightwarning' ) )
-					.find( 'a' ).attr( 'target', '_blank' ).end()
-			);
+ve.init.wikia.ViewPageTarget.prototype.shouldReloadPageAfterSave = function () {
+	return this.userLoggedInDuringEdit ||
+		ve.init.wikia.ViewPageTarget.super.prototype.shouldReloadPageAfterSave.call( this );
+};
+
+ve.init.wikia.ViewPageTarget.prototype.getVeNotifyAfterSave = function () {
```

Comment:
This method looks like more universal than in parent class. Maybe we could move it there and don't override it in child class?